src/frontend/components/router-tree/router-tree.ts

```diff
@@ -75,11 +76,13 @@ export class RouterTree {
       .attr('class', (d) => d.isAux ? 'node-aux-route' : 'node-route')
       .attr('r', 6);
 
+    const fillColor: string = this.theme === 'dark' ? '#A5A5A5' : '#000000';
```

Comment:
Is there any way we could avoid hardcoding colors inside the JavaScript? I believe you can set svg fill from inside a css block